cmd/lakectl/cmd/store/s3.go

```diff
@@ -31,6 +31,10 @@ func (s *S3Walker) Walk(ctx context.Context, storageURI *url.URL, walkFn func(e
 	var continuation *string
 	const maxKeys = 1000
 	prefix := strings.TrimLeft(storageURI.Path, "/")
+	var trimPrefix string
+	if idx := strings.LastIndex(prefix, "/"); idx != -1 {
+		trimPrefix = prefix[:idx+1]
```

Comment:
It took me some time to understand why this is the right value, it's based on the `ObjectStoreEntry` definition:
```
	// RelativeKey represents a path relative to prefix (or directory). If none specified, will be identical to FullKey
```

Would you mind adding a comment explaining the `trimPrefix` part?

Reply:
Done